subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyInsightReportTask.java

```diff
@@ -94,9 +105,11 @@
 @DisableCachingByDefault(because = "Produces only non-cacheable console output")
 public class DependencyInsightReportTask extends DefaultTask {
 
+    private final Set<SelectionReasonOutput> enabledSelectionReasonOutputs = EnumSet.noneOf(SelectionReasonOutput.class);
```

Comment:
I'd drop the "enabled" from the name of this and the accessors.  Makes me think this is a boolean.